ClientApp/src/Pages/SubmitSighting/SubmitSighting.scss

```diff
@@ -0,0 +1,39 @@
+﻿@import "../../Styles/constants";
+
+.submit-sighting-page {
+  margin: $gutter-width;
+  
+  .title {
+    @include titleFont;
+  }
+}
+
+.submit-sighting-form {
+  position: relative;
+  left: 50%;
+  transform: translateX( -50%);
+
+  .form-label {
+    @include formLabel;
+  }
+  
+  .form-input {
+    @include formInput;
+  }
+  
+  .submit-button { 
+    @include primaryButton;
+  }
+}
+
+
+
+
+@media only screen and (max-width: 600px) {
+.submit-sighting.form{
+  position: relative;
+  width: 90%;
+  left: 50%;
```

Comment:
these will already be inherited from above so we don't need to specify them again